src/BusinessNotificationCenter.jsx

```diff
@@ -0,0 +1,120 @@
+import './App.css';
+import { useEffect, useState } from 'react';
+import { useBackend } from './contexts/BackendContext';
+import { SearchIcon, PlusSquareIcon, BellIcon, HamburgerIcon } from '@chakra-ui/icons';
+import {
+  Text,
+  InputGroup,
+  InputLeftElement,
+  Icon,
+  Menu,
+  MenuButton,
+  Button,
+  Input,
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+} from '@chakra-ui/react';
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  ModalCloseButton,
+  useDisclosure,
+} from '@chakra-ui/react';
+
+const App = () => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const { backend } = useBackend();
+  const [data, setData] = useState([]);
+
+  const getData = async () => {
+    try {
+      const id = '5';
+      const response = await backend.get(`/notification/${id}`);
+      setData(response.data);
+      console.log(data);
+    } catch (error) {
+      console.error('Error fetching data:', error);
+    }
+  };
+
+  useEffect(() => {
+    getData();
+  }, []);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'getData'. Either include it or remove the dependency array.

Reply:
use a comment to tell eslint to shut up (hover over the yellow squiggy thing to get the autogenerated comment for exhaustive-deps